File: app/src/main/java/com/example/myappcontacts/presentation/contacts/presenter/IContactsPresenter.java
package com.example.myappcontacts.presentation.contacts.presenter;

import com.example.myappcontacts.data.dao.contacts.db.ContactsModel;

import java.util.UUID;

public interface IContactsPresenter {
    void loadContact(UUID contactId);

    void updateContact(ContactsModel contactsModel);

    void onEditSaveButtonClicked();

    void onPhotoImageClicked();

    void photoUriLoaded(String photoUri);
}

File: app/src/main/java/com/example/myappcontacts/presentation/contacts/presenter/IContactsPresenter.kt
package com.example.myappcontacts.presentation.contacts.presenter

import com.example.myappcontacts.data.dao.contacts.db.ContactsModel
import java.util.*

interface IContactsPresenter {

    fun loadContact(contactId: UUID)

    fun updateContact(contactsModel: ContactsModel)

    fun onEditSaveButtonClicked()

    fun onPhotoImageClicked()

    fun photoUriLoaded(photoUri: String)
}